dirty_cat/_super_vectorizer.py

```diff
@@ -180,31 +184,31 @@ class SuperVectorizer(ColumnTransformer):
         Note that using this feature requires that the DataFrame columns
         input at :term:`fit` and :term:`transform` have identical order.
 
-    sparse_threshold: float, default=0.3
+    sparse_threshold : float, optional, default=0.3
         If the output of the different transformers contains sparse matrices,
         these will be stacked as a sparse matrix if the overall density is
         lower than this value. Use sparse_threshold=0 to always return dense.
         When the transformed output consists of all dense data, the stacked
         result will be dense, and this keyword will be ignored.
 
-    n_jobs : int, default=None
+    n_jobs : int, optional, default=None
```

Comment:
Why did you leave here `default=None` here and removed it everywhere else?

Reply:
Good point! There's actually a specific reason I left it, and it's because it's explicitly mentioned in the `n_jobs` doc (just below) that `None means 1 unless in a joblib.parallel_backend context.`, so essentially it's a way to indicate to the user "by default, it's 1".

Reply:
Looking back at the numpy style guide, I noticed this is not the expected way of doing it, so I modified it :)